ide-common/src/main/kotlin/org/digma/intellij/plugin/posthog/ActivityMonitor.kt

```diff
@@ -229,6 +226,13 @@ class ActivityMonitor(private val project: Project) /*: Runnable, Disposable*/ {
         )
     }
 
+    fun registerContainerEngine(containerPlatform: String?) {
```

Comment:
please avoid the nulls.
use within the enum of `ContainerEngine` the NONE and/or UNKOWN

Reply:
removed "?"